test/Eto.Test/Sections/Controls/DocumentControlSection.cs

```diff
@@ -77,6 +77,13 @@ Control SelectPage()
 			return control;
 		}
 
+		Control ShowNavigationStepper()
+		{
+			var control = new CheckBox { Text = "Show navigation buttons" };
+			control.Bind(box => box.Checked, tabControl, documentControl => documentControl.HasNavigationButtons);
```

Comment:
Would be nice to align the naming in this method based on `HasNavigationButtons` property.

Reply:
Renamed

Reply:
Renamed to `AllowNavigationButtons`